src/plugins/walletConnectToDapps/components/modals/SessionProposal.tsx

```diff
@@ -27,10 +29,10 @@ const checkAllNamespacesSupported = (
 ): boolean => {
   return Object.values(namespaces).every(
     namespace =>
-      namespace.chains?.every(chainId =>
-        // TODO(gomes): fix this
-        walletSupportsChain({ chainId, wallet, isSnapInstalled: false }),
-      ),
+      namespace.chains?.every(chainId => {
+        const chainAccountIds = selectAccountIdsByChainId(store.getState(), { chainId })
```

Comment:
same thing, as it stands there is no reactivity here

Reply:
This one is fine, this isn't a hook but a pure function

Reply:
84414deb62

Reply:
Actually very smort - 740d7f196